homeassistant/components/trafikverket_train/sensor.py

```diff
@@ -50,7 +50,7 @@
                 vol.Required(CONF_NAME): cv.string,
                 vol.Required(CONF_TO): cv.string,
                 vol.Required(CONF_FROM): cv.string,
-                vol.Optional(CONF_TIME): cv.time,
+                vol.Optional(CONF_TIME): cv.string,
```

Comment:
Please leave the schema in-tact and convert it before passing it for import or handle it at import